packages/evm/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 1.2.2 - 2022-10-26
+
+- Fixed `EIP-3540` bug where an EOF header combination was not rejected when calling, PR [#2381](https://github.com/ethereumjs/ethereumjs-monorepo/pull/2381)
```

Comment:
```suggestion
- Fixed `EIP-3540` bug where EOF header validation was incorrectly applied to legacy contract code in EVM calls, PR [#2381](https://github.com/ethereumjs/ethereumjs-monorepo/pull/2381)
```

I think the issue here was we were treating non EOF code (i.e. contract bytecode that didn't start with EOF) as EOF code.  Right, @jochem-brouwer ?

Reply:
Ok, applied.